financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/settings/PlaygroundSettings.kt

```diff
@@ -0,0 +1,136 @@
+package com.stripe.android.financialconnections.example.settings
+
+import android.app.Application
+import android.content.Context
+import android.util.Log
+import androidx.core.content.edit
+import com.stripe.android.financialconnections.example.BuildConfig
+import com.stripe.android.financialconnections.example.data.model.LinkAccountSessionBody
+import com.stripe.android.financialconnections.example.data.model.PaymentIntentBody
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+
+internal data class PlaygroundSettings(
+    val settings: Map<PlaygroundSettingDefinition<*>, Any?>
+) {
+    fun <T> withValue(
```

Comment:
I think this is where things differ from the payments variant. This makes PlaygroundSettings immutable (kinda), but makes every change require a full copy.

Reply:
yeah, wanted to stay in line with state immutability. All these functions are just helpers to generate inmutable settings copies with updated values.